services/ai-navigator-cluster-info-agent/0.1.2/defaults/cm.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: ai-navigator-cluster-info-agent-0.1.1-d2iq-defaults
+  name: ai-navigator-cluster-info-agent-0.1.2-d2iq-defaults
   namespace: ${releaseNamespace}
 data:
   values.yaml: |
```

Comment:
Let's rollback all the changes for `ai-navigator-cluster-info-agent` since they are not needed for the cluster info API patches